docs/install.rst

```diff
@@ -68,17 +84,32 @@ activating your virtualenv:
 
 .. code-block:: bash
 
-  # . /opt/pyff/bin/activate
+  # source python-pyff/bin/activate
 
 Next install pyFF:
 
 .. code-block:: bash
 
-  # pip install pyFF
+  # cd pyFF
+  # LANG=en_US.UTF-8 pip install -e .
 
 This will install a bunch of dependencies and compile bindings for both lxml, pyyaml
-aswell as pyXMLSecurity. This may take some time to complete. If there are no errors and if
-you have the *pyff* binary in your **$PATH** you should be done.
+as well as pyXMLSecurity. This may take some time to complete. If there are no errors and if
+you have the *python-pyff* binary in your **$PATH** you should be done.
+
+.. code-block:: bash
+
+ # cd $HOME
+ # mkdir pyff-config
+ # cd pyff-config
+
+pyFF requires five files which you should copy into the pyff-config directory
+
+- A stylesheet in the *xsl* format
+- A metadata signer certficate
+- A metadata signer key 
+- A pipeline file in the *yaml* format
+- A *xrd* file 
```

Comment:
xrd files are also not necessary - its just a way to represent a list of metadata URLs and their acoompanying signing certs but you could also do a load statement with multiple URL references... or a mix of xrd and URLs. There is also an (undocumented) way parser for so call MDSL files which are used in the eIDAS federation